packages/core/saved-objects/core-saved-objects-migration-server-internal/src/kibana_migrator.ts

```diff
@@ -39,11 +39,12 @@ import { DocumentMigrator } from './document_migrator';
 import { runZeroDowntimeMigration } from './zdt';
 import { ALLOWED_CONVERT_VERSION } from './kibana_migrator_constants';
 import { runV2Migration } from './run_v2_migration';
-
+import * as HASH_TO_VERSION_MAP from './hash_to_version_map.json';
```

Comment:
NIT: why is the `import * as XXX` required here compared to `import XXX from` ?

Reply:
It was not apparently. I have moved it to `constants.ts` on `@kbn/core-saved-objects-base-server-internal`.